djpsa/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-VERSION = (0, 8, '0a0')
+VERSION = (0, 8, '0a1')
```

Comment:
Not that it makes a difference at all, but the bit after "a" is typically the build number- indicating a change in something about the build process, but the result should be the same as what was built before it.

So this should be changed to "1a0", not "0a1", if we want to be accurate. Or just just 8 to 9, since it's not a bugfix.